apix_documentation_swagger_v4.yaml

```diff
@@ -0,0 +1,1186 @@
+                    next: null
+                    prev: null
+
+  /challenges/{id}:
+    patch:
+      tags:
+        - Challenges
+      summary: Replies to a challenge
+      description: |
+        Sends the response to a specific challenge
+      operationId: updateChallenge
+      parameters:
+        - name: id
+          in: path
+          description: ID of the challenge that is being answered
+          required: true
+          schema:
+            type: string
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                challenge:
+                  type: object
+                  required:
+                    - answer
+                  properties:
+                    answer:
+                      type: string
+
+      responses:
+        200:
+          description: successful response
+          content:
+            application/vnd.regalii.v4.0+json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: object
+                    oneOf:
+                      - $ref: '#/components/schemas/MFA_Challenge_Question'
+                      - $ref: '#/components/schemas/MFA_Challenge_Code'
+                      - $ref: '#/components/schemas/MFA_Challenge_Image'
+                      - $ref: '#/components/schemas/MFA_Challenge_Image_Choices'
+                      - $ref: '#/components/schemas/MFA_Challenge_Choices'
+
+components:
+  securitySchemes:
+    hmacAuth:
+      type: http
+      scheme: hmac-sha1
+  schemas:
+    GenericData:
+      type: object
+      required:
+        - type
+        - id
+      properties:
+        type:
+          type: string
+          description: Describes the type of data that is being returned
+        id:
+          type: string
+          description: Unique identifier (UUID) of the entity
+
+    Merchant:
+      type: object
+      required:
+        - attributes
+      allOf:
+        - $ref: '#/components/schemas/GenericData'
+      properties:
+        attributes:
+          type: object
+          required:
+            - name
+            - country
+            - currency
+            - created_at
+            - updated_at
+          properties:
+            name:
+              type: string
+              description: Merchant's name
+            country:
+              type: string
+              description: Merchant’s ISO 2 digit country code
+            currency:
+              type: string
+              description: 3 digit ISO currency code of the merchant’s
+                country
+            category:
+              type: string
+              description: Short description of the merchant's
+                category (e.g. Television/Internet, electricity, etc.)
+            home_url:
+              type: string
+              description: ""
```

Comment:
Missing descriptions of `*_url`

Reply:
@GRoguelon I added some of them but I'm not sure they are accurate. I left some TODOs because I did not know what to write in some cases.

Reply:
Can a task be created in Jira with this?